test/behandlerdialog/MeldingTilBehandlerTest.tsx

```diff
@@ -100,10 +106,12 @@ describe("MeldingTilBehandler", () => {
       expect(screen.getByText(legeerklaringText)).to.exist;
     });
 
-    it("Viser radiobuttons med behandlervalg, der det ikke er mulig å velge 'Ingen behandler'", () => {
+    it("Viser radiobuttons med behandlervalg, der det ikke er mulig å velge 'Ingen behandler'", async () => {
       renderMeldingTilBehandler();
 
-      expect(screen.queryByText("Ingen behandler")).to.not.exist;
+      await waitFor(
+        () => expect(screen.queryByText("Ingen behandler")).to.be.null
+      );
```

Comment:
Viste seg at det var litt vanskelig å sjekke at ting ikke finnes når formet oppfører seg asynkront. Si ifra hvis noen har forslag for å gjøre dette penere.